test/unit/rules/lint-aria-unsupported-elements-test.js

```diff
@@ -0,0 +1,78 @@
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'aria-unsupported-elements',
+
+  config: true,
+
+  good: ['<div role="button"></div>', '<img role="option">'],
```

Comment:
Can you add `good` examples using the prohibited elements also?

Reply:
done